### PR TITLE
frontend, backend: run rawhide-to-release for all CoprDirs

### LINKED ISSUE #3016: f39: Giving up waiting for copr_base repository
My project fails the builds for f39,
whereas all other builds are fine.
Here's the log:
https://download.copr.fedorainfracloud.org/results/stsp/dosemu2/fedora-39-x86_64/06679713-dosemu2/backend.log.gz
Its always the same.
What does this mean?

### PR DESCRIPTION
**Pull request by FrostyX**
Fix #3016

### CODE REVIEW

File: frontend/coprs_frontend/commands/rawhide_to_release.py [hunk 1]
```diff
@@ -82,7 +83,7 @@ def rawhide_to_release_function(rawhide_chroot, dest_chroot, retry_forked):
                 func.max(models.Build.id),
             )
             .join(models.BuildChroot)
-            .filter(models.Build.copr_dir_id == copr.main_dir.id)
+            .filter(models.Build.copr_id == copr.id)
```
Comment (praiskup) [inline frontend/coprs_frontend/commands/rawhide_to_release.py:86]: I think you are querying the last Build in each Copr, not each CoprDir.

File: frontend/coprs_frontend/tests/test_commands/test_rawhide_to_release.py [hunk 2]
```diff
@@ -144,17 +144,17 @@ def test_branch_fedora(self, capsys):
...
             "Handling builds in copr 'user1/foocopr', chroot 'fedora-rawhide-x86_64'",
-            "Createrepo for copr 'user1/foocopr', chroot 'fedora-19-x86_64'",
+            "Createrepo for 'user1/foocopr', chroot 'fedora-19-x86_64'",
             "fedora-19-i386 - already exists.",
             "fedora-19-x86_64 - already exists.",
             "Handling builds in copr 'user2/barcopr', chroot 'fedora-rawhide-i386'",
             "  Fresh new build chroots: 0, regenerate 1",
             "Handling builds in copr 'user1/foocopr', chroot 'fedora-rawhide-x86_64'",
-            "Createrepo for copr 'user1/foocopr', chroot 'fedora-19-x86_64'",
+            "Createrepo for 'user1/foocopr', chroot 'fedora-19-x86_64'",
```
Comment (praiskup) [inline frontend/coprs_frontend/tests/test_commands/test_rawhide_to_release.py:158]: Would you mind editing the `test_branch_fedora` so it tests some custom CoprDir, too?

**Comment by praiskup** [comment]
Test would be nice, but LGTM!

**Comment by FrostyX** [comment]
Adding tests for custom chroots sounds like a good idea, I will do that.

**Comment by FrostyX** [comment]
I added some tests, PTAL. It is in a separate commit for an easier review, I will squash them before merging.

**Review by praiskup — APPROVED** [review]
+1

**Pull request merged**